userguide/system.rst

```diff
@@ -442,63 +442,63 @@ The buttons above the boot entries can be used to:
 #ifdef freenas
 .. figure:: images/be2.png
 
-   Viewing the Status of the Boot Device
+   Viewing the Status of the |OS-Device|
 #endif freenas
 #ifdef truenas
 .. figure:: images/tn_be2.png
 
-   Viewing the Status of the Boot Device
+   Viewing the Status of the |OS-Device|
 #endif truenas
 
 
 #ifdef freenas
 If the system has a mirrored boot pool, there will be a
 :guilabel:`Detach` button in addition to the :guilabel:`Replace` button.
 To remove a device from the boot pool, highlight the device and click
-its :guilabel:`Detach` button. Alternately, if one of the boot devices
+its :guilabel:`Detach` button. Alternately, if one of the |os-devices|
 has an *OFFLINE* :guilabel:`Status`, click the device to replace,
 then click :guilabel:`Replace` to rebuild the boot mirror.
 #endif freenas
 #ifdef truenas
-If one of the boot devices has a :guilabel:`Status` of *OFFLINE*,
+If one of the |os-devices| has a :guilabel:`Status` of *OFFLINE*,
 click the device to replace, select the new replacement device, and
 click :guilabel:`Replace Disk` to rebuild the boot mirror.
 #endif truenas
 
 #ifdef freenas
 Note that
-**the boot device cannot be replaced if it is the only boot device**
+**the |os-device| cannot be replaced if it is the only |os-device|**
 because it contains the operating system itself.
 #endif freenas
 
 
 #ifdef freenas
-.. index:: Mirroring the Boot Device
-.. _Mirroring the Boot Device:
+.. index:: Mirroring the |OS-Device|
+.. _Mirroring the |OS-Device|:
 
-Mirroring the Boot Device
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Mirroring the |OS-Device|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If the system is currently booting from a device, another device
```

Comment:
paragraph should say "If the system is currently booting from a single |os-device|, another device can be added to create a mirrored |os-device|. If one device.."